sweetercat/plot.py

```diff
@@ -194,6 +197,8 @@ def plot_page(df, columns, request, page):
     css_resources = INLINE.render_css()
 
     script, div = components(layout)
+    if error is not None:
```

Comment:
If you changed the error to be `True / False` only then you would only need `if error:`

Any reason for `True / None`?